fandogh_cli/certificate_commands.py

```diff
@@ -0,0 +1,49 @@
+import click
+from fandogh_cli.fandogh_client.certificate_client import detail_certificate, create_certificate
+from .utils import format_text, TextStyle
+from .base_commands import FandoghCommand
+import time
+
+
+@click.group("ssl")
```

Comment:
why isn't is a part of secret? 
couldn't it be a new type of secrets?

Reply:
SSL certificates are `Certificate` objects in Kubernetes, and they are not directly equivalent to secrets.
even if we assume certificates are secrets, this would cause secret API to become too complicated.
If you insist on not having another group for SSL then I suggest moving `ssl create` under domain command and merging `ssl details` with `domain details`

Reply:
@MahdiZareie OK, let's chat about it